truelearn/datasets/__init__.py

```diff
@@ -8,8 +14,10 @@
 )
 
 __all__ = [
+    "check_and_download_file",
```

Comment:
we should not expose these.

Reply:
You dont want to expose Remote file meta data either?

Reply:
> You dont want to expose Remote file meta data either?

Yes. Because these are for internal uses.